src/progressive_automations_python/prefect_flows.py

```diff
@@ -0,0 +1,235 @@
+"""
+Simplified Prefect flows for automated desk control.
+
+Provides scheduled automation and workflow orchestration using Prefect.
+Directly decorates core functions from desk_controller for task execution.
+"""
+
+import time
+from prefect import flow, task
+from prefect.logging import get_run_logger
+
+from progressive_automations_python.desk_controller import (
+    move_to_height, 
+    check_duty_cycle_status_before_execution
+)
+from progressive_automations_python.testing import (
+    test_sequence,
+    execute_custom_movements
+)
+from progressive_automations_python.config import LOWEST_HEIGHT
+from progressive_automations_python.duty_cycle import get_duty_cycle_status, load_state
+
+# Decorate core functions as tasks
+move_to_height_task = task(move_to_height)
```

Comment:
Does it make sense to decorate them here functionally rather than to simply add @task to the original? I suppose we could leave it this way, since that way prefect isn't a strict dependency for the other code running.